src/main/java/io/vertx/proton/ProtonClientOptions.java

```diff
@@ -394,4 +395,34 @@ public ProtonClientOptions setHeartbeat(int heartbeat) {
   public int getHeartbeat() {
     return this.heartbeat;
   }
+
+  /**
+   * Sets the maximum frame size to announce in the AMQP <em>OPEN</em> frame.
+   * <p>
+   * If this property is not set explicitly, a reasonable default value is used.
+   * <p>
+   * Setting this property to -1 will result in no maximum frame size being announced at all.
```

Comment:
Perhaps say "negative" rather than "to -1" to match what it appears the impl does?